apps/app.near/widget/Components/Pixel/Button.jsx

```diff
@@ -0,0 +1,74 @@
+const StyledButton = styled.div`
+  all: unset;
+  display: flex;
+  padding: 8px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+
+  border-radius: 8px;
+  border: ${(props) => {
+    switch (props.variant) {
+      case "transparent":
+        return "none";
+      default:
+        return "1px solid var(--stroke);";
+    }
+  }};
+  background: ${(props) => {
+    switch (props.variant) {
+      case "primary":
+        return "var(--btn-primary-bg)";
+      case "transparent":
+        return "var(--bg2)";
+      default:
+        return "var(--bg1)";
+    }
+  }};
+
+  /* Pixel/Shadow/Light/Tag & Button */
+  box-shadow: ${(props) => {
```

Comment:
Why is there a Pixel.Button, Pixel.FollowButton, Pixel.Hashtag, Pixel.LinkTree... 

Can there be a generic Button, Hashtag, etc?

What is Profile.FollowButton and Profile.LinkTree?

Reply:
The button styling is different for both themes, that's why we have different Buttons and stuff.
The ones in Profile are common

Reply:
What might be some solutions for combining? 

What if a common <Button still handled things like variants, but not other customizations like border, hover effects, etc.... But it did have a "button" class applied to it, and this ".button" was customized in the theme root?

Reply:
That could work, but both themes have different set of variants. I will have to start from ground up lol.